google-cloud-storage/lib/google/cloud/storage/bucket.rb

```diff
@@ -2249,7 +2249,7 @@ def patch_gapi! *attributes
           patch_args = Hash[attributes.map do |attr|
             [attr, @gapi.send(attr)]
           end]
-          patch_gapi = API::Bucket.new patch_args
+          patch_gapi = API::Bucket.new **patch_args
```

Comment:
lib/google/cloud/storage/bucket.rb:2258:40: W: Lint/AmbiguousOperator: Ambiguous keyword splat operator. Parenthesize the method arguments if it's surely a keyword splat operator, or add a whitespace to the right of the ** if it should be a exponent.

```suggestion
patch_gapi = API::Bucket.new(**patch_args)
```